FirebaseCore/Internal/Sources/HeartbeatLogging/HeartbeatsPayload.swift

```diff
@@ -91,6 +91,12 @@ extension HeartbeatsPayload: HTTPHeaderRepresentable {
   public func headerValue() -> String {
     let encoder = JSONEncoder()
     encoder.dateEncodingStrategy = .formatted(Self.dateFormatter)
+    #if DEBUG
+      if #available(iOS 11, *) {
```

Comment:
Note: [`sortedKeys`](https://developer.apple.com/documentation/foundation/jsonencoder/outputformatting/2919670-sortedkeys) is only available on iOS 11.0+, macOS 10.13+, tvOS 11.0+, watchOS 4.0+, visionOS 1.0+. [`FirebaseCoreInternal`](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCoreInternal.podspec#L21-L24) is already macOS 10.13+, tvOS 12.0+, watchOS 6.0+ so I just specified `iOS 11` in the availability check and let the rest fall through with `*`.

Reply:
Added a TODO to remind us to remove the `if #available` check when we increase our deployment target to iOS 11+.